configure.ac

```diff
@@ -2087,7 +2087,7 @@ if test -z "${BOLT_APPLY_FLAGS}"
 then
   AS_VAR_SET(
     [BOLT_APPLY_FLAGS],
-    [m4_join([ ],
+    ["m4_join([ ],
```

Comment:
This looks awkward to me; it is better Autoconf practice to use:

```m4
    [m4_normalize("
      -update-debug-sections
      -reorder-blocks=ext-tsp
      ...
    ")]

Reply:
If this suggestion is applied, I prefer this option to #104751

Reply:
> This looks awkward to me; it is better Autoconf practice to use:

Yeah agree.